service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/PractitionerAgentPersonMapperTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import static org.mockito.Mockito.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.hl7.fhir.dstu3.model.Organization;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.hl7.fhir.dstu3.model.PractitionerRole;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import uk.nhs.adaptors.gp2gp.common.service.FhirParseService;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+import uk.nhs.adaptors.gp2gp.utils.ResourceTestFileUtils;
+import uk.nhs.adaptors.gp2gp.utils.TestArgumentsLoaderUtil;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class PractitionerAgentPersonMapperTest {
+
+    private static final String TEST_ID = "6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73";
+    private static final String PRACTITIONER_FILE_LOCATION = "/ehr/mapper/practitioner/";
+    private static final String PRACTITIONER_ONLY_FILE_LOCATION = PRACTITIONER_FILE_LOCATION + "practitioner-only/";
+    private static final String PRACTITIONER_ROLE_FILE_LOCATION = PRACTITIONER_FILE_LOCATION + "practitioner-role/";
+    private static final String ORGANIZATION = PRACTITIONER_FILE_LOCATION + "organization.json";
+    private static final String PRACTITIONER = PRACTITIONER_FILE_LOCATION + "practitioner.json";
+
+    private PractitionerAgentPersonMapper practitionerAgentPersonMapper;
+    @Mock
+    private RandomIdGeneratorService randomIdGeneratorService;
```

Comment:
Could we move the mocked objects to the top or bottom of the field list? To make a clear separation between mocked and not mocked fields.